core/src/main/java/org/elasticsearch/common/xcontent/XContentBuilder.java

```diff
@@ -70,6 +69,8 @@ public static XContentBuilder builder(XContent xContent, Set<String> includes, S
 
     private boolean humanReadable = false;
 
+    private TimeUnit timeUnit = TimeUnit.MILLISECONDS;
```

Comment:
I think it might be worth dropping the `timeUnit` for now? I'm just scared of it.